swabbie-front50/src/test/kotlin/com/netflix/spinnaker/swabbie/front50/ApplicationResourceOwnerResolutionStrategyTest.kt

```diff
@@ -32,23 +33,42 @@ object ApplicationResourceOwnerResolutionStrategyTest {
   private val subject = ApplicationResourceOwnerResolutionStrategy(front50ApplicationCache)
 
   @Test
-  fun `should resolve resource owner from application config`() {
+  fun `should resolve resource owner from spinnaker application`() {
     whenever(front50ApplicationCache.get()) doReturn
       setOf(
         Application(name = "name", email = "name@netflix.com"),
         Application(name = "test", email = "test@netflix.com")
       )
 
-    subject.resolve(R()) shouldMatch equalTo("name@netflix.com")
+    subject.resolve(ResourceOne()) shouldMatch equalTo("name@netflix.com")
+  }
+
+  @Test
+  fun `should resolve multiple resource owner from spinnaker application`() {
```

Comment:
Since we assume applications are distinct in spinnaker, and remove duplicate applications, I'm going to keep this test as is. I may add an example 'other' owner provider that demos adding other ownership, but not sure yet.

Reply:
Very reasonable